hpat/tests/test_series.py

```diff
@@ -1711,6 +1712,35 @@ def test_impl(S):
         S = pd.Series(np.random.ranf(m))
         self.assertEqual(hpat_func(S), test_impl(S))
 
+    def test_series_median_skipna_default1(self):
+        '''Verifies median implementation with default skipna=True argument on a series with NA values'''
+        def test_impl(S):
+            res = S.median()
+            print(res)
```

Comment:
Please remove `print` if it was used for debugging. I don't think we need that here.

Reply:
Thank you.